frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -172,7 +172,7 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
     type,
   });
 
-  const resultDataVolumeName = prefixedID(vmName, name);
+  const resultDataVolumeName = generateDataVolumeName(vmName, name);
```

Comment:
can you please write a comment here for future archeologists? Something akin to
```
we can generate a random name every time, because this modal should not operate on disks with live datavolumes
```

Reply:
Done! I think your comment is perfect just like it is, so I didn't change it.